Gemfile.lock

```diff
@@ -82,80 +82,80 @@ GEM
     crass (1.0.6)
     erubi (1.9.0)
     execjs (2.7.0)
-    ffi (1.12.2)
+    ffi (1.13.1)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
-    i18n (1.8.2)
+    i18n (1.8.5)
       concurrent-ruby (~> 1.0)
     jbuilder (2.10.0)
       activesupport (>= 5.0.0)
-    jquery-rails (4.3.5)
+    jquery-rails (4.4.0)
```

Comment:
### [jquery-rails](https://github.com/rails/jquery-rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.3.5 → 4.4.0

[_~~change-log~~, [source-code](https://github.com/rails/jquery-rails)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/rails/jquery-rails/compare/v4.3.5...v4.4.0).

These are the individual commits:
- (b6d6e25) [Fix `unescape_js` to match changes in Action View](https://github.com/rails/jquery-rails/commit/b6d6e2514889ad576c23b8783127e0c346db0d43)
- (6361d43) [Merge pull request #269 from repinel/fix-unescape-js](https://github.com/rails/jquery-rails/commit/6361d43027345c391731c9f403ed1a51b02ea467)
- (ad3227e) [Test against latest Ruby versions](https://github.com/rails/jquery-rails/commit/ad3227ef6536687ad8a7b756822551f630e3aaab)
- (90826fd) [Bump jQuery version to 3.5.1](https://github.com/rails/jquery-rails/commit/90826fdc955c830db19307d1085e08bd3ebf9c28)
- (a8b003d) [Release v4.4.0](https://github.com/rails/jquery-rails/commit/a8b003d726522cf663611c114d8f0e79abf8d200)


</details>